classes/class-bsf-custom-fonts-render.php

```diff
@@ -228,6 +228,16 @@ public function add_block_assets_style() {
 			if ( is_admin() && ! is_customize_preview() ) {
 				add_action( 'enqueue_block_assets', array( $this, 'add_style' ) );
 			}
+			add_action( 'enqueue_block_editor_assets', array( $this, 'enque_data' ) );
+		}
+
+		/**
+		 * Enqueue iFrame Preview Script.
+		 *
+		 * @since 1.3.7
+		 */
+		public function enque_data() {
+			wp_enqueue_script( 'bsf-custom-block-js', BSF_CUSTOM_FONTS_URI . 'assets/js/custom-fonts-preview.js', array( 'jquery' ), '2', true );
```

Comment:
1. As you have written in plain JS code no `'jquery'` dependency needed here. Keep simple blank array.
2. Add proper plugin's constant version number instead of '2'

Reply:
Sure! I will update this.

Reply:
Comment Resolved.